### PR TITLE
UVES tutorial formatting & Learning Goals

### PR DESCRIPTION
**Pull request by emilylurice**
I updated the header of the tutorial to match the Coordinates format and added Learning Goals

@eblur @bsipocz

### CODE REVIEW

File: tutorials/notebooks/UVES/UVES.ipynb [hunk 1]
```diff
@@ -7,6 +7,39 @@
...
+    "* *Coding*: Make code reusable as a function\n",
+    "* *Analysis*: Analyze spectral data using [`astropy.units`](http://docs.astropy.org/en/stable/units/index.html),  [`astropy.constants`](http://docs.astropy.org/en/stable/constants/index.html), and [`astropy.time`](http://docs.astropy.org/en/stable/time/index.html)\n",
+    "* *Input/Output*: Generate a LaTeX table using [`astropy.table`](http://docs.astropy.org/en/stable/table/index.html)\n",
+    "* *Visualization*: Plot spectral data using [`matplotlib`](https://matplotlib.org/) \n",
+    "\n",
+    "## Keywords:\n",
+    "astropy.io, astropy.wcs, astropy.units, astropy.constants, astropy.time, astropy.table, matplotlib, spectra, UVES\n",
+    "\n",
+    "This tutorial follows my real live data analysis of MN Lup and the code developed\n",
+    "below is taken (with only minor modifications) from the code that I used to\n",
```
Comment (bsipocz) [inline tutorials/notebooks/UVES/UVES.ipynb]: use "we"/"our" instead of "I"/"my"

**Comment by kelle** [comment]
Emily, we're working on changing test infrastructure right now. When we get that sorted out, we'll work on helping getting this pull request merged.

In the meantime, if you want to try to clear outputs: open the notebook, Click Kernels > Restart Kernel and Clear all outputs. Then save the notebook and commit and push the change.

**Comment by adrn** [comment]
I went ahead and rebased this PR to get the new build machinery in to this PR, and cleared the output along the way. Thanks @emilurice! I think this will be ready once the tests pass

**Comment by eblur** [comment]
Rebased and text merged with recent Astropy Tutorial copy edits.

Needs copy editing on the new text: just the first two cells of text.

**Comment by bsipocz** [comment]
@eblur - it looks like the rebase gone wrong, there are now tons of unrelated commits in here (I suspect the merges are the culprits). I'm happy to help out if needed, if you ping me about it in a few days on slack?

**Comment by eblur** [comment]
I re-ran the tests and it runs fine. Looks like we had a hiccup in server access for some of the tutorials, not anything to do with this PR. Merging!

**Comment by adrn** [comment]
@eblur in the future, it'd be good to wait until someone signs off (i.e. "accepts" the PR) before merging! Something weird happened here with the rebase, and I now need to make sure nothing got over-written accidentally

**Pull request merged**